axelrod/strategy_transformers.py

```diff
@@ -141,6 +153,27 @@ def __repr__(self):
                     prefix = ', '
                 return name
 
+            def reduce_for_decorated_class(self_):
```

Comment:
I'm not terribly sure to be honest. I'm happy as is, I'm guessing that quote does not necessarily apply to nested def and class?